cmd/login/runner_config_modify_test.go

```diff
@@ -268,6 +268,7 @@ func TestKubeConfigModification(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
+		tc := tc
```

Comment:
How come is this now necessary?

Reply:
Lint error of implicit memory aliasing. Newer Go versions finally get rid of this by making the loop variables locally-scoped, but for now we need this. My changes didn't lead to the build errors, but I wanted to fix them.